File: C/problem1.c
#include "stdio.h"
#include "time.h"

int main(int argc, char* argv[]) {
    clock_t start, end;
    double cpu_time_used;
    int sum;

    start = clock();

    sum = 0;

    for (int i = 1; i < 1000; i++) {
        if ((i % 3 == 0) || (i % 5 == 0)) {
            sum = sum + i;
        }
    }

    printf("%d\n", sum);

    end = clock();
    cpu_time_used = ((double) (end - start)) / CLOCKS_PER_SEC;

    printf("%f seconds\n", cpu_time_used);

    return 0;
}

File: python/problem1.py
''' If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.

Find the sum of all the multiples of 3 or 5 below 1000. '''

import time

start = time.time()

summ = 0

for i in range(1, 1000):
	if (i % 3 == 0) or (i % 5 == 0):
		summ += i

print(summ)

print("Operation took " + str(time.time() - start) + " seconds")


'''
> python3 problem1.py 
233168
Operation took 0.03300142288208008 seconds

'''